src/components/network/equipment-table.js

```diff
@@ -1,13 +1,25 @@
-import React, { useEffect, useRef, useState } from 'react';
+import React, { useCallback, useEffect, useRef, useState } from 'react';
 import LoaderWithOverlay from '../util/loader-with-overlay';
 import { MultiGrid, AutoSizer } from 'react-virtualized';
 import {
     MIN_COLUMN_WIDTH,
     HEADER_ROW_HEIGHT,
     ROW_HEIGHT,
 } from './config-tables';
+import makeStyles from '@mui/styles/makeStyles';
+
+const useStyles = makeStyles(() => ({
+    emptyLastLineCell: {
+        margin: '5px',
```

Comment:
use theme.spacing(1) or 2, or 3 ... not raw px value

Reply:
Ok done.